File: DBtoCSharp_MARIADB/modelos/INNODB_METRICS.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    public class INNODB_METRICS
    {
            [Required]
    [StringLength(193)]
        /// <summary>
        /// 
        /// </summary>
        public string NAME { get; set; }
            [Required]
    [StringLength(193)]
        /// <summary>
        /// 
        /// </summary>
        public string SUBSYSTEM { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long COUNT { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long MAX_COUNT { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long MIN_COUNT { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public float AVG_COUNT { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long COUNT_RESET { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long MAX_COUNT_RESET { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long MIN_COUNT_RESET { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public float AVG_COUNT_RESET { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public string TIME_ENABLED { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public string TIME_DISABLED { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public long TIME_ELAPSED { get; set; }
        
        /// <summary>
        /// 
        /// </summary>
        public string TIME_RESET { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int ENABLED { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public string TYPE { get; set; }
            [Required]
    [StringLength(193)]
        /// <summary>
        /// 
        /// </summary>
        public string COMMENT { get; set; }

}
}

File: DBtoCSharp_MARIADB/output/INNODB_METRICS.ts
export interface INNODB_METRICS
    {
        /**///  */
        NAME: string;
        /**///  */
        SUBSYSTEM: string;
        /**///  */
        COUNT: number;
        /**///  */
        MAX_COUNT: number;
        /**///  */
        MIN_COUNT: number;
        /**///  */
        AVG_COUNT: number;
        /**///  */
        COUNT_RESET: number;
        /**///  */
        MAX_COUNT_RESET: number;
        /**///  */
        MIN_COUNT_RESET: number;
        /**///  */
        AVG_COUNT_RESET: number;
        /**///  */
        TIME_ENABLED: string;
        /**///  */
        TIME_DISABLED: string;
        /**///  */
        TIME_ELAPSED: number;
        /**///  */
        TIME_RESET: string;
        /**///  */
        ENABLED: number;
        /**///  */
        TYPE: string;
        /**///  */
        COMMENT: string;
}